gui/src/main/java/org/jboss/as/console/client/shared/subsys/undertow/HttpMetricPresenter.java

```diff
@@ -178,7 +179,7 @@ public void loadDetails() {
 
         ModelNode operation = new ModelNode();
         operation.get(OP).set(READ_RESOURCE_OPERATION);
-        operation.get(ADDRESS).set(Baseadress.get());
+        operation.get(ADDRESS).set(new ResourceAddress().add("host", serverStore.getSelectedServer().getHostName()).add("server", serverStore.getSelectedServer().getServerName()));
```

Comment:
It did when I tested it. Standalone seems to go through a different code path. I will double check it later today to be 100% sure.